src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -1093,7 +1093,10 @@ const generateExitDiagnostics = (diag) => {
 };
 
 const extractRuntimeOutput = (podCompletionStatus) => {
-  if (_.isEmpty(podCompletionStatus)) {
+  if (
+    _.isEmpty(podCompletionStatus) ||
+    !_.isArray(podCompletionStatus.containers)
```

Comment:
why not use built-in `Array.isArray` in es6

Reply:
Good to known that

Reply:
Fixed